NUTranslator/time.js

```diff
@@ -0,0 +1,4 @@
+export default (value) => {
+  const d = new Date(parseInt(value));
+  return d.toLocaleString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true,second:'numeric' });
```

Comment:
hmm! here too, we need to be able to translate dates based on the user's locale! Again open a bug for this as well! Will address it in 5.4